.github/workflows/update_docs.yml

```
@@ -0,0 +1,30 @@
+
+on:
+  push:
+    branches:
+      - main
+      - add_docs
+
+jobs:
+  pages:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
```

actionlint (1.7.4)

property "deployment" is not defined in object type {}

(expression)